share/picongpu/examples/LaserWakefield/lib/python/picongpu/ui_params.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
can we call this file just `params.py` or with the also included `.cfg` options `input.py`?

Because we could also set all other options exposed in its `share/picongpu/examples/LaserWakefield/cmakeFlags` file (and the .cfg file).

Reply:
@codingS3b please still rename the file